frontend/src/customPropTypes/noticeOfWork.js

```diff
@@ -7,8 +7,152 @@ export const nowApplication = shape({
   mine_guid: PropTypes.string,
   mine_name: PropTypes.string,
   minenumber: PropTypes.string,
-  noticeofworktype: PropTypes.string,
-  trackingnumber: PropTypes.number,
+  trackingnumber: PropTypes.string,
```

Comment:
The API is setup with `'trackingnumber': fields.Integer,`, so this should be a number

Reply:
oops, Didn't intend to delete that one